backend/users/forms.py

```diff
@@ -58,3 +64,41 @@ def save(self):
         user = authenticate(email=email, password=password)
         login(self.context, user)
         return user
+
+
+class UpdateForm(ContextAwareModelForm):
+    date_birth = CharField(required=False)
+    open_to_recruiting = BooleanField(required=False)
+    open_to_newsletter = BooleanField(required=False)
+    country = CharField(required=False)
+
+    def save(self, commit=True):
+        super().save(commit=commit)
+        return User.objects.get(id=self.context.user.id)
+
+    class Meta:
+        model = User
+        fields = ("first_name", "last_name", "gender")
+
+
+class UpdateImageForm(FormWithContext):
```

Comment:
Why the image has a different form mutation?

Maybe you can check that if the `image` is specified in the form's payload you can run the code you are now running in the other form?

Just an idea

Reply:
Yes, true. I thought to divide because usually in the FE the update of the image is done separately, but in the BE is only more code...

Reply:
done in [e4d51ce](https://github.com/pythonitalia/pycon/pull/788/commits/e4d51ce74487f20cc4089e100cb2d06cb8b6e864)